gtsfm/three_view_estimator.py

```diff
@@ -0,0 +1,110 @@
+"""Estimator which operates on a triplet of images to compute relative pose and verified indices.
+
+Authors: Ayush Baid, John Lambert
+"""
+import logging
+
+import dask
+import numpy as np
+from dask.delayed import Delayed
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.frontend.matcher.matcher_base import MatcherBase
+from gtsfm.frontend.verifier.verifier_base import VerifierBase
+
+logger = logger_utils.get_logger()
+
+mpl_logger = logging.getLogger("matplotlib")
```

Comment:
thanks for the PR, Ayush.

Maybe we should configure the matplotlib logger settings inside `get_logger()`? are they needed here?